src/js/edu-benefits/1990-rjsf/BenefitsRelinquishmentField.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash/fp';
+
+/**
+ * Field for selecting the benefitsRelinquished and corresponding date as necessary.
+ * @type {Object}
+ */
+export default class BenefitsRelinquishmentField extends React.Component {
+  onPropertyChange(name) {
+    return (value) => {
+      const formData = Object.keys(this.props.formData || {}).length
+        ? this.props.formData
+        : {
```

Comment:
It's probably not necessary, since there's no way to make a form data change on this page that removes all properties.